src/hosted.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * NOTE: this is meant for debugging Bramble, and as an example
+ * of how to do basic things with the API. It wasn't intended as a production setup.
+ *
+ * Use ?forceFiles=1 to force startup to re-install the default files
+ */
+(function() {
+    "use strict";
+
+    require.config({
+        baseUrl: './'
+    });
+
+    // Allow the user to override what we do with default files using `?forceFiles=1`
+    var forceFiles = window.location.search.indexOf("forceFiles=1") > -1;
+
+    // Default filesystem content
+    var projectRoot = "/7/projects/30";
+
+    var index = "<html>\n"                                  +
```

Comment:
Yeah.  I stopped worrying about this when ropes got added to JS impls, http://blog.cdleary.com/2012/01/string-representation-in-spidermonkey/.  I also sometimes let my MAKEFILE-writing self shine through and use multiline in JS :) http://davidwalsh.name/multiline-javascript-strings